### ISSUE TITLE
Accessibility Audit for Top Task pages

### ISSUE THREAD

**Issue by ContentBrewmaster** [department-of-veterans-affairs/va.gov-cms#10041]
**1. What is this request for?**

This request is to assess the accessibility for published content related Register for care, Billing and insurance and Medical records tasks that are not using the new nodes created for these tasks in Q2 2022.

**2. What is the primary function and/or purpose of the product?**

These pages represent key tasks performed by Veterans when engaging with their local VAMC. 

**3. Are there any known accessibility issues that have not yet been addressed? If so, please add GitHub links to those issues below.**

A known issue is the use of the phrase "coming soon" on detail pages for these tasks. While not a traditional accessibility issue, use of this language does diminish the Veteran experience. We suspect that since editors have an open text field to present this content, other accessibility issues may exist.
  
**4. Are there any specific testing requirements that should be noted?**

No
  
**5. Please provide anything needed to access and review the product.**
  
Billing and insurance: https://prod.cms.va.gov/admin/content/audit?title_op=contains&title=Billing+and&type_op=not+in&type%5B%5D=vamc_system_billing_insurance&moderation_state_op=in&section=All
Medical records: https://prod.cms.va.gov/admin/content/audit?title_op=contains&title=medical+record&type_op=not+in&type%5B%5D=vamc_system_medical_records_offi&moderation_state_op=in&moderation_state%5B%5D=editorial-published&section=8
Register for care: https://prod.cms.va.gov/admin/content/audit?title_op=contains&title=register+for&type_op=not+in&type%5B%5D=vamc_system_medical_records_offi&moderation_state_op=in&moderation_state%5B%5D=editorial-published&section=8
  
**6. When does the product need to launch?**

The top task pages are not scheduled for a launch. This audit will help VHA Digital Media determine next steps to have existing top task content migrate to the new nodes for greater consistency and a better experience for Veterans.

**Comment by EWashb** [comment]
@dawnpruitt this will fall into unplanned work for our team for Sprint 64 (if we get it completed)

**Comment by EWashb** [comment]
After talking with PO, this work is not a priority for this sprint. We have a general idea of the accessibility concerns around top tasks, however, there is nothing introduced from an accessibility standpoint by noncompliance. There were some issues found with headers in our CMS design though and those have been ticketed: #10060 #10062

**Comment by EWashb** [comment]
This is not needed
